plugins/org.eclipse.papyrus.uml.diagram.sequence.runtime/src/org/eclipse/papyrus/uml/diagram/sequence/runtime/internal/edit/policies/ISequenceEditPolicy.java

```diff
@@ -54,7 +54,7 @@ default LayoutHelper getLayoutHelper() {
 	}
 
 	default LayoutConstraints getLayoutConstraints() {
-		return getLayoutHelper().getConstraints();
+		return Activator.getDefault().getLayoutConstraints(__getEditingDomain(this));
```

Comment:
Letting clients access the `LayoutConstraints` directly from the activator allows us to decouple it from the `LayoutHelper` in the future.